app/views/patient_mailer/assessment_email.html.erb

```diff
@@ -17,21 +17,22 @@
           text: t('assessments.email.reminder.report', locale: @lang)
         } %><br />
 
-    <% if @contact_info[:email].present? || @contact_info[:phone].present? || @contact_info[:webpage].present? %>
-      <p><%= t('assessments.email.contact.info1', locale: @lang) %></p>
-      <% if @contact_info[:email].present? %>
-        <%= t('assessments.email.contact.email', locale: @lang) %><%= ':'%> <%= @contact_info[:email] %> <br />
+    <% if @patients.count == index + 1 %>
+      <% if @contact_info[:email].present? || @contact_info[:phone].present? || @contact_info[:webpage].present? %>
```

Comment:
would it be possible to put this block outside of the patients loop to avoid querying the database again every iteration by calling `count`?

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/948/commits/741f2a7795fe12df1b22baecb351900e1a655bb4